jekyll/_layouts/classic-docs.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 {% include head.html %}
-<body class="{% if page.path == "index.html" %}homepage{% endif %}">
+<body class="{% if page.page-type == "index" or page.page-type == "homepage" %}homepage{% endif %}">
```

Comment:
This shouldn't be necessary. The old index page, should be converted into using `layout: index-page`. The new one needs it's own layout file. We can do it after the PR though.